packages/py-moose-lib/moose_lib/data_models.py

```
@@ -668,11 +696,19 @@ def _to_columns(model: type[BaseModel]) -> list[Column]:
+        )
+
+        # Validate mutual exclusivity of DEFAULT, MATERIALIZED, and ALIAS
+        modifier_count = sum(
+            1 for v in [default_expr, materialized_expr, alias_expr] if v is not None
+        )
+        if modifier_count > 1:
             raise ValueError(
-                f"Column '{column_name}' cannot have both DEFAULT and MATERIALIZED. "
-                f"Use one or the other."
+                f"Column '{column_name}' can only have one of DEFAULT, MATERIALIZED, or ALIAS."
             )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)